htcanalyze/htcanalyze.py

```diff
@@ -904,26 +836,18 @@ def analyzed_summary(self, log_files: list_of_logs) -> log_inf_list:
                         n_host_nodes['Aborted before'
                                      ' submission'] = [1, total_time]
 
-                    # convert nan values to 0
-                    if res_dict:
-                        res_dict["Usage"] = np.nan_to_num(res_dict["Usage"])
-                        res_dict["Requested"] = np.nan_to_num(
-                            res_dict["Requested"])
-                        res_dict["Allocated"] = np.nan_to_num(
-                            res_dict["Allocated"])
-
                     all_files[term_type] = [1,
                                             waiting_time,
                                             runtime,
                                             total_time,
-                                            res_dict,
+                                            [resource_list],
```

Comment:
Isn't `resource_list` already a list? Is there a reason for making it a `List[list]`? And if it's not a list, why is it called `resource_list`?

Reply:
I get that this is strange, but yeah the nested lists are wanted here, cause every log file has normally 3 resources, which I store in a list, these therefore are saved in a bigger list, to calculate the average.
I think you would prefer a kind of JobResources class, to make this more obvious. 
I think a typing definition would make obvious enough without adding to much.

Reply:
I renamed the `resources` to `job_resources`, so that `[job_resources]` makes more sense. As this is a list of state related job resources. I think that should be obvious enough.